popup.js

```diff
@@ -64,7 +64,7 @@ const modalData = [
     },
     image: './images/snap3.png',
     description:
-      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
```

Comment:
- [ ] Also as suggested by the previous reviewer and rightly so, the buttons for your live and source links should display without an overflow to scroll down. This is in line with best practices for web accessibility and gives the best user experience 👏 
- Seeing that your design is great, I suggest you reduce the content of the `loresum` wording and possible overflow, so the buttons show on the first display. good job so far 👍 

# View now:
![image](https://user-images.githubusercontent.com/50990925/216736475-fbf3065f-291b-4a29-9a46-2d7b86420598.png)

# Figma reommnndation :
![image](https://user-images.githubusercontent.com/50990925/216736501-38033cf7-2186-467e-ad2c-7386a4bf1669.png)